packages/dx-react-grid-bootstrap3/src/templates/table-select-row.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const TableSelectRow = ({
+  tableRow,
+  children,
+  changeSelected,
+  selectByRowClick,
+  ...restProps
+}) => (
+  <tr
+    className={tableRow.selected ? 'active' : ''}
+    onClick={selectByRowClick ? (e) => {
```

Comment:
I think that when this option is not enabled, we should avoid rendering that template. It may unexpectedly override custom user template. So, we should add a note to the docs that rowTemplate for data rows defined in TableView, will be overriden by selectRowTemplate when the property is true.